lab/lab1/part2-fizzbuzz.html

```diff
@@ -26,7 +26,14 @@
     Start code
 
     ===================== */
-
+    for (var i = 1; i <= 100; i = i+1) {
+      if (i%3 == 0 && i%5 != 0) {
+    console.log('Fizz');}
```

Comment:
The results should look like this:

![image](https://user-images.githubusercontent.com/8103418/52545832-8fa0e800-2d88-11e9-802c-791380d7a10f.png)


Yours look like this:

![image](https://user-images.githubusercontent.com/8103418/52545819-7ac45480-2d88-11e9-83f2-2b289b85d7e2.png)



You need to print out the numbers not divisible by 3 or 5 also